### PR TITLE
Bump Substrate and Deps

### PR DESCRIPTION
**Pull request by adoerr**
Bump Substrate in order to get [Fix transactions not being propagated to authorities](https://github.com/paritytech/substrate/commit/ad626b00990e57f7d23dee212d3b78e758ae88c8) fixed.

### CODE REVIEW

**Comment by adoerr** [comment]
Had to add a couple of `#[allow(clippy::manual_map)]`.

This check seems to be quite new and still has issues. See [manual_map recommends noncompilable code involving moved values](lhttps://github.com/rust-lang/rust-clippy/issues/6797).

**Comment by adoerr** [comment]
WTF, was all green, requested the reviews and then it turned read

**Comment by adoerr** [comment]
PR has been tested with a local test net and looked good, AFAICT.

**Comment by HCastano** [comment]
I tested the builds locally and they're fine. 

There's probably an issue with the CI runners, @TriplEight, would you be able to take a look at this on Monday?

File: relays/ethereum/src/main.rs [hunk 1]
```diff
@@ -243,6 +243,8 @@ fn ethereum_deploy_contract_params(matches: &clap::ArgMatches) -> Result<Ethereu
 	let eth_contract_code = parse_hex_argument(matches, "eth-contract-code")?.unwrap_or_else(|| {
 		hex::decode(include_str!("../res/substrate-bridge-bytecode.hex")).expect("code is hardcoded, thus valid; qed")
 	});
+
+	#[allow(clippy::manual_map)]
```
Comment (HCastano) [inline relays/ethereum/src/main.rs:247]: Can you open an issue so that we can address this once the lint is working correctly?

**Review by HCastano — APPROVED** [review]

File: relays/ethereum/src/main.rs (hunk 1 above)
Comment (adoerr) [reply relays/ethereum/src/main.rs:247]: #773

**Pull request merged**